README.md

```diff
@@ -1,76 +1,30 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Position and Floating
 
-> One paragraph statement about the project.
+Project #3 Cloning a NY Times article
 
-![screenshot](./app_screenshot.png)
+![screenshot](./app_screenshot.png/)
 
 Additional description about the project and its features.
 
 ## Built With
 
-- Major languages,
-- frameworks,
-- technologies used
+- HTML,
+- CSS,
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://cvilla714.github.io/floating/)
 
+👤 **kc**
 
-## Getting Started
+- Github: [@githubhandle](https://github.com/cvilla714)
+- Twitter: [@twitterhandle](https://twitter.com/kckeyti)
+- Linkedin: [linkedin](https://www.linkedin.com/in/cosmel-villalobos-1900531aa/)
```

Comment:
inside the [  ] add your social medias. e.g.: Instead of @githubhandle add your @username